packages/service-library/src/servicelib/monitoring.py

```diff
@@ -58,6 +59,10 @@ async def middleware_handler(request: web.Request, handler):
                 resp_time,
                 resp.status,
             )
+        except asyncio.CancelledError as exc:
+            # python 3.8 cancellederror is not an Exception
```

Comment:
rephrase this note: in py3.8: CancelledError is now a subclass of BaseException and NOT Exception